### PR TITLE
Admin: Add support to configure default admin email 

### PR DESCRIPTION
**Pull request by mhuangwm**
**What this PR does / why we need it:**
Implements admin_email config option. 

**Which issue(s) this PR fixes:**
Fixes #52336

### CODE REVIEW

File: pkg/setting/setting.go [hunk 1]
```diff
@@ -1248,6 +1249,7 @@ func readSecuritySettings(iniFile *ini.File, cfg *Cfg) error {
 	cfg.DisableInitAdminCreation = security.Key("disable_initial_admin_creation").MustBool(false)
 	cfg.AdminUser = valueAsString(security, "admin_user", "")
 	cfg.AdminPassword = valueAsString(security, "admin_password", "")
+	cfg.AdminEmail = valueAsString(security, "admin_email", "")
```
Comment (mgyongyosi) [inline pkg/setting/setting.go:1252]: I think using `cfg.AdminUser + "@localhost"` as the fallback/default value would be reasonable, what do you think @Jguer?

**Review by mgyongyosi — CHANGES_REQUESTED** [review]
Thank you for your PR @mhuangwm! 💯  It looks good!  I left a small change request and I'd like to ask you to extend the `conf/sample.ini` file with this new configuration option too.

File: pkg/setting/setting.go (hunk 1 above)
Comment (Jguer) [reply pkg/setting/setting.go:1252]: :+1: I agree, it would also avoid breaking the previous default behavior

Comment (sakjur) [reply pkg/setting/setting.go:1252]: I believe that `defaults.ini` will take precedence and set the email to `admin@localhost` even when `admin_user` has changed, so there'd be a ever so slight modification of current behavior.

File: conf/defaults.ini [hunk 2]
```diff
@@ -247,6 +247,9 @@ admin_user = admin
 # default admin password, can be changed before first start of grafana, or in profile settings
 admin_password = admin
 
+# default admin email, created on startup
+admin_email = admin@localhost
```
Comment (sakjur) [inline conf/defaults.ini:251]: Please update `sample.ini` as well.

File: pkg/setting/setting.go (hunk 1 above)
Comment (sakjur) [reply pkg/setting/setting.go:1252]: To be clear, I think that's fine and nothing to stress about, it's just good to be aware that that might happen to someone.

**Review by mgyongyosi — APPROVED** [review]
Thanks again @mhuangwm for your PR! 💯  It looks good to me!

**Pull request merged**